// 登陆成功
9月 06 12:59:41 vm login[5945]: pam_unix(login:session): session opened for user yl by LOGIN(uid=0)
9月 06 12:59:41 vm systemd[1]: Started Session 17 of user yl.
9月 06 12:59:41 vm systemd-logind[635]: New session 17 of user yl.

-----------------------

9月 06 12:59:41 vm login[5945]: pam_unix(login:session): session opened for user yl by LOGIN(uid=0)

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_CAP_EFFECTIVE=1ffffffffff(26)
_TRANSPORT=syslog(17)
SYSLOG_FACILITY=10(18)
_AUDIT_LOGINUID=1000(20)
SYSLOG_IDENTIFIER=login(23)
*MESSAGE=pam_unix(login:session): session opened for user yl by LOGIN(uid=0)(75)
_COMM=login(11)
_EXE=/usr/bin/login(19)
*_CMDLINE=/bin/login -p --(25)
_SYSTEMD_CGROUP=/system.slice/system-getty.slice/getty@tty3.service(67)
*_SYSTEMD_UNIT=getty@tty3.service(32)
_SYSTEMD_SLICE=system-getty.slice(33)
_AUDIT_SESSION=17(17)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
SYSLOG_PID=5945(15)
SYSLOG_TIMESTAMP=Sep  6 12:59:41 (33)
_PID=5945(9)
_SYSTEMD_INVOCATION_ID=d2c934047002402caaf08c39a98a083e(55)
_SOURCE_REALTIME_TIMESTAMP=1693976381178464(43)
-----------------------

9月 06 12:59:41 vm systemd[1]: Started Session 17 of user yl.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
SYSLOG_FACILITY=3(17)
SYSLOG_IDENTIFIER=systemd(25)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_TRANSPORT=journal(18)
_CAP_EFFECTIVE=1ffffffffff(26)
_PID=1(6)
_COMM=systemd(13)
_EXE=/usr/lib/systemd/systemd(29)
_CMDLINE=/sbin/init(19)
_SYSTEMD_CGROUP=/init.scope(27)
_SYSTEMD_UNIT=init.scope(24)
_SYSTEMD_SLICE=-.slice(22)
CODE_FILE=src/core/job.c(24)
CODE_LINE=929(13)
CODE_FUNC=job_log_done_status_message(37)
JOB_TYPE=start(14)
JOB_RESULT=done(15)
MESSAGE_ID=39f53479d3a045ac8e11786248231fbf(43)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
MESSAGE=Started Session 17 of user yl.(38)
JOB_ID=8971(11)
UNIT=session-17.scope(21)
INVOCATION_ID=8a7a9e29e7d140aba62a8454dec0589b(46)
_SOURCE_REALTIME_TIMESTAMP=1693976381183070(43)
-----------------------

9月 06 12:59:41 vm systemd-logind[635]: New session 17 of user yl.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_SYSTEMD_SLICE=system.slice(27)
_TRANSPORT=journal(18)
SYSLOG_FACILITY=4(17)
SYSLOG_IDENTIFIER=systemd-logind(32)
_COMM=systemd-logind(20)
_EXE=/usr/lib/systemd/systemd-logind(36)
_CMDLINE=/lib/systemd/systemd-logind(36)
_CAP_EFFECTIVE=24420020f(24)
_SYSTEMD_CGROUP=/system.slice/systemd-logind.service(52)
_SYSTEMD_UNIT=systemd-logind.service(36)
CODE_FILE=src/login/logind-session.c(36)
CODE_LINE=698(13)
CODE_FUNC=session_start(23)
MESSAGE_ID=8d45620c1a4348dbb17410da57c60c66(43)
USER_ID=yl(10)
_PID=635(8)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
_SYSTEMD_INVOCATION_ID=625f7cbfebca48f287836c8c01a0e21d(55)
SESSION_ID=17(13)
LEADER=5945(11)
MESSAGE=New session 17 of user yl.(34)
_SOURCE_REALTIME_TIMESTAMP=1693976381183222(43)
-----------------------


// 退出
9月 06 12:59:52 vm systemd[1]: Started Getty on tty6.
9月 06 12:59:52 vm login[5945]: pam_unix(login:session): session closed for user yl
9月 06 12:59:52 vm systemd[1]: getty@tty3.service: Succeeded.
9月 06 12:59:52 vm systemd[1]: session-17.scope: Succeeded.
9月 06 12:59:52 vm systemd-logind[635]: Session 17 logged out. Waiting for processes to exit.
9月 06 12:59:52 vm systemd[1]: getty@tty3.service: Scheduled restart job, restart counter is at 1.
9月 06 12:59:52 vm systemd[1]: Stopped Getty on tty3.
9月 06 12:59:52 vm systemd[1]: Started Getty on tty3.
9月 06 12:59:52 vm systemd-logind[635]: Removed session 17.

-----------------------

9月 06 12:59:52 vm systemd[1]: Started Getty on tty6.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
SYSLOG_FACILITY=3(17)
SYSLOG_IDENTIFIER=systemd(25)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_TRANSPORT=journal(18)
_CAP_EFFECTIVE=1ffffffffff(26)
_PID=1(6)
_COMM=systemd(13)
_EXE=/usr/lib/systemd/systemd(29)
_CMDLINE=/sbin/init(19)
_SYSTEMD_CGROUP=/init.scope(27)
_SYSTEMD_UNIT=init.scope(24)
_SYSTEMD_SLICE=-.slice(22)
CODE_FILE=src/core/job.c(24)
CODE_LINE=929(13)
CODE_FUNC=job_log_done_status_message(37)
JOB_TYPE=start(14)
JOB_RESULT=done(15)
MESSAGE_ID=39f53479d3a045ac8e11786248231fbf(43)
MESSAGE=Started Getty on tty6.(30)
UNIT=getty@tty6.service(23)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
JOB_ID=9057(11)
INVOCATION_ID=093d712eb6eb4abe9c6ece22fbdeb9d6(46)
_SOURCE_REALTIME_TIMESTAMP=1693976392643833(43)
-----------------------

9月 06 12:59:52 vm login[5945]: pam_unix(login:session): session closed for user yl

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
_UID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_CAP_EFFECTIVE=1ffffffffff(26)
_TRANSPORT=syslog(17)
SYSLOG_FACILITY=10(18)
_GID=1000(9)
_AUDIT_LOGINUID=1000(20)
_SYSTEMD_OWNER_UID=1000(23)
_SYSTEMD_SLICE=user-1000.slice(30)
_SYSTEMD_USER_SLICE=-.slice(27)
SYSLOG_IDENTIFIER=login(23)
_COMM=login(11)
_EXE=/usr/bin/login(19)
*_CMDLINE=/bin/login -p --(25)
MESSAGE=pam_unix(login:session): session closed for user yl(59)
_AUDIT_SESSION=17(17)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
SYSLOG_PID=5945(15)
_PID=5945(9)
SYSLOG_TIMESTAMP=Sep  6 12:59:52 (33)
_SYSTEMD_CGROUP=/user.slice/user-1000.slice/session-17.scope(60)
_SYSTEMD_SESSION=17(19)
_SYSTEMD_UNIT=session-17.scope(30)
_SYSTEMD_INVOCATION_ID=8a7a9e29e7d140aba62a8454dec0589b(55)
_SOURCE_REALTIME_TIMESTAMP=1693976392644717(43)
-----------------------

9月 06 12:59:52 vm systemd[1]: getty@tty3.service: Succeeded.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
SYSLOG_FACILITY=3(17)
SYSLOG_IDENTIFIER=systemd(25)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_TRANSPORT=journal(18)
_CAP_EFFECTIVE=1ffffffffff(26)
CODE_FILE=src/core/unit.c(25)
CODE_LINE=5870(14)
CODE_FUNC=unit_log_success(26)
MESSAGE_ID=7ad2d189f7e94e70a38c781354912448(43)
_PID=1(6)
_COMM=systemd(13)
_EXE=/usr/lib/systemd/systemd(29)
_CMDLINE=/sbin/init(19)
_SYSTEMD_CGROUP=/init.scope(27)
_SYSTEMD_UNIT=init.scope(24)
_SYSTEMD_SLICE=-.slice(22)
UNIT=getty@tty3.service(23)
MESSAGE=getty@tty3.service: Succeeded.(38)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
INVOCATION_ID=d2c934047002402caaf08c39a98a083e(46)
_SOURCE_REALTIME_TIMESTAMP=1693976392652414(43)
-----------------------

9月 06 12:59:52 vm systemd[1]: session-17.scope: Succeeded.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
SYSLOG_FACILITY=3(17)
SYSLOG_IDENTIFIER=systemd(25)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_TRANSPORT=journal(18)
_CAP_EFFECTIVE=1ffffffffff(26)
CODE_FILE=src/core/unit.c(25)
CODE_LINE=5870(14)
CODE_FUNC=unit_log_success(26)
MESSAGE_ID=7ad2d189f7e94e70a38c781354912448(43)
_PID=1(6)
_COMM=systemd(13)
_EXE=/usr/lib/systemd/systemd(29)
_CMDLINE=/sbin/init(19)
_SYSTEMD_CGROUP=/init.scope(27)
_SYSTEMD_UNIT=init.scope(24)
_SYSTEMD_SLICE=-.slice(22)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
UNIT=session-17.scope(21)
INVOCATION_ID=8a7a9e29e7d140aba62a8454dec0589b(46)
MESSAGE=session-17.scope: Succeeded.(36)
_SOURCE_REALTIME_TIMESTAMP=1693976392654516(43)
-----------------------

9月 06 12:59:52 vm systemd-logind[635]: Session 17 logged out. Waiting for processes to exit.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_SYSTEMD_SLICE=system.slice(27)
_TRANSPORT=journal(18)
SYSLOG_FACILITY=4(17)
SYSLOG_IDENTIFIER=systemd-logind(32)
_COMM=systemd-logind(20)
_EXE=/usr/lib/systemd/systemd-logind(36)
_CMDLINE=/lib/systemd/systemd-logind(36)
_CAP_EFFECTIVE=24420020f(24)
_SYSTEMD_CGROUP=/system.slice/systemd-logind.service(52)
_SYSTEMD_UNIT=systemd-logind.service(36)
CODE_FILE=src/login/logind-session.c(36)
USER_ID=yl(10)
CODE_LINE=770(13)
CODE_FUNC=session_stop_scope(28)
_PID=635(8)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
_SYSTEMD_INVOCATION_ID=625f7cbfebca48f287836c8c01a0e21d(55)
SESSION_ID=17(13)
LEADER=5945(11)
MESSAGE=Session 17 logged out. Waiting for processes to exit.(61)
_SOURCE_REALTIME_TIMESTAMP=1693976392656869(43)
-----------------------

9月 06 12:59:52 vm systemd[1]: getty@tty3.service: Scheduled restart job, restart counter is at 1.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
SYSLOG_FACILITY=3(17)
SYSLOG_IDENTIFIER=systemd(25)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_TRANSPORT=journal(18)
_CAP_EFFECTIVE=1ffffffffff(26)
_PID=1(6)
_COMM=systemd(13)
_EXE=/usr/lib/systemd/systemd(29)
_CMDLINE=/sbin/init(19)
_SYSTEMD_CGROUP=/init.scope(27)
_SYSTEMD_UNIT=init.scope(24)
_SYSTEMD_SLICE=-.slice(22)
UNIT=getty@tty3.service(23)
CODE_FILE=src/core/service.c(28)
CODE_LINE=2296(14)
CODE_FUNC=service_enter_restart(31)
MESSAGE_ID=5eb03494b6584870a536b337290809b3(43)
MESSAGE=getty@tty3.service: Scheduled restart job, restart counter is at 1.(75)
N_RESTARTS=1(12)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
INVOCATION_ID=d2c934047002402caaf08c39a98a083e(46)
_SOURCE_REALTIME_TIMESTAMP=1693976392657377(43)
-----------------------

9月 06 12:59:52 vm systemd[1]: Stopped Getty on tty3.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
SYSLOG_FACILITY=3(17)
SYSLOG_IDENTIFIER=systemd(25)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_TRANSPORT=journal(18)
_CAP_EFFECTIVE=1ffffffffff(26)
_PID=1(6)
_COMM=systemd(13)
_EXE=/usr/lib/systemd/systemd(29)
_CMDLINE=/sbin/init(19)
_SYSTEMD_CGROUP=/init.scope(27)
_SYSTEMD_UNIT=init.scope(24)
_SYSTEMD_SLICE=-.slice(22)
CODE_FILE=src/core/job.c(24)
CODE_LINE=929(13)
CODE_FUNC=job_log_done_status_message(37)
JOB_RESULT=done(15)
MESSAGE_ID=9d1aaa27d60140bd96365438aad20286(43)
UNIT=getty@tty3.service(23)
MESSAGE=Stopped Getty on tty3.(30)
JOB_TYPE=restart(16)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
INVOCATION_ID=d2c934047002402caaf08c39a98a083e(46)
JOB_ID=9142(11)
_SOURCE_REALTIME_TIMESTAMP=1693976392657638(43)
-----------------------

9月 06 12:59:52 vm systemd[1]: Started Getty on tty3.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
SYSLOG_FACILITY=3(17)
SYSLOG_IDENTIFIER=systemd(25)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_TRANSPORT=journal(18)
_CAP_EFFECTIVE=1ffffffffff(26)
_PID=1(6)
_COMM=systemd(13)
_EXE=/usr/lib/systemd/systemd(29)
_CMDLINE=/sbin/init(19)
_SYSTEMD_CGROUP=/init.scope(27)
_SYSTEMD_UNIT=init.scope(24)
_SYSTEMD_SLICE=-.slice(22)
CODE_FILE=src/core/job.c(24)
CODE_LINE=929(13)
CODE_FUNC=job_log_done_status_message(37)
JOB_TYPE=start(14)
JOB_RESULT=done(15)
MESSAGE_ID=39f53479d3a045ac8e11786248231fbf(43)
MESSAGE=Started Getty on tty3.(30)
UNIT=getty@tty3.service(23)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
JOB_ID=9142(11)
INVOCATION_ID=326397d40c2c4a9a91a7d33d2647a6c1(46)
_SOURCE_REALTIME_TIMESTAMP=1693976392661931(43)
-----------------------

9月 06 12:59:52 vm systemd-logind[635]: Removed session 17.

_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
PRIORITY=6(10)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_SYSTEMD_SLICE=system.slice(27)
_TRANSPORT=journal(18)
SYSLOG_FACILITY=4(17)
SYSLOG_IDENTIFIER=systemd-logind(32)
_COMM=systemd-logind(20)
_EXE=/usr/lib/systemd/systemd-logind(36)
_CMDLINE=/lib/systemd/systemd-logind(36)
_CAP_EFFECTIVE=24420020f(24)
_SYSTEMD_CGROUP=/system.slice/systemd-logind.service(52)
_SYSTEMD_UNIT=systemd-logind.service(36)
CODE_FILE=src/login/logind-session.c(36)
USER_ID=yl(10)
CODE_LINE=827(13)
CODE_FUNC=session_finalize(26)
MESSAGE_ID=3354939424b4456d9802ca8333ed424a(43)
_PID=635(8)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
_SYSTEMD_INVOCATION_ID=625f7cbfebca48f287836c8c01a0e21d(55)
SESSION_ID=17(13)
LEADER=5945(11)
MESSAGE=Removed session 17.(27)
_SOURCE_REALTIME_TIMESTAMP=1693976392666452(43)
-----------------------


// 用户名正确，密码错误
9月 06 13:05:08 vm login[6071]: pam_unix(login:auth): authentication failure; logname=LOGIN uid=0 euid=0 tty=/dev/tty3 ruser= rhost=  user=yl
9月 06 13:05:12 vm login[6071]: FAILED LOGIN (1) on '/dev/tty3' FOR 'yl', Authentication failure

-----------------------

9月 06 13:05:08 vm login[6071]: pam_unix(login:auth): authentication failure; logname=LOGIN uid=0 euid=0 tty=/dev/tty3 ruser= rhost=  user=yl

PRIORITY=5(10)
_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_CAP_EFFECTIVE=1ffffffffff(26)
_TRANSPORT=syslog(17)
SYSLOG_FACILITY=10(18)
SYSLOG_IDENTIFIER=login(23)
_COMM=login(11)
_EXE=/usr/bin/login(19)
_CMDLINE=/bin/login -p --(25)
_SYSTEMD_CGROUP=/system.slice/system-getty.slice/getty@tty3.service(67)
_SYSTEMD_UNIT=getty@tty3.service(32)
_SYSTEMD_SLICE=system-getty.slice(33)
MESSAGE=pam_unix(login:auth): authentication failure; logname=LOGIN uid=0 euid=0 tty=/dev/tty3 ruser= rhost=  user=yl(117)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
SYSLOG_PID=6071(15)
SYSLOG_TIMESTAMP=Sep  6 13:05:08 (33)
_PID=6071(9)
_SYSTEMD_INVOCATION_ID=ecf7f5332b17411e892a9162e0a801d0(55)
_SOURCE_REALTIME_TIMESTAMP=1693976708888966(43)
-----------------------

9月 06 13:05:12 vm login[6071]: FAILED LOGIN (1) on '/dev/tty3' FOR 'yl', Authentication failure

PRIORITY=5(10)
_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_CAP_EFFECTIVE=1ffffffffff(26)
_TRANSPORT=syslog(17)
SYSLOG_FACILITY=10(18)
SYSLOG_IDENTIFIER=login(23)
_COMM=login(11)
_EXE=/usr/bin/login(19)
_CMDLINE=/bin/login -p --(25)
_SYSTEMD_CGROUP=/system.slice/system-getty.slice/getty@tty3.service(67)
_SYSTEMD_UNIT=getty@tty3.service(32)
_SYSTEMD_SLICE=system-getty.slice(33)
MESSAGE=FAILED LOGIN (1) on '/dev/tty3' FOR 'yl', Authentication failure(72)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
SYSLOG_PID=6071(15)
_PID=6071(9)
_SYSTEMD_INVOCATION_ID=ecf7f5332b17411e892a9162e0a801d0(55)
SYSLOG_TIMESTAMP=Sep  6 13:05:12 (33)
_SOURCE_REALTIME_TIMESTAMP=1693976712550635(43)
-----------------------


// 用户名和密码都错误
9月 06 13:06:23 vm login[6074]: pam_unix(login:auth): check pass; user unknown
9月 06 13:06:23 vm login[6074]: pam_unix(login:auth): authentication failure; logname=LOGIN uid=0 euid=0 tty=/dev/tty3 ruser= rhost=
9月 06 13:06:26 vm login[6074]: FAILED LOGIN (1) on '/dev/tty3' FOR 'UNKNOWN', Authentication failure

-----------------------

9月 06 13:06:23 vm login[6074]: pam_unix(login:auth): check pass; user unknown

PRIORITY=5(10)
_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_CAP_EFFECTIVE=1ffffffffff(26)
_TRANSPORT=syslog(17)
SYSLOG_FACILITY=10(18)
SYSLOG_IDENTIFIER=login(23)
_COMM=login(11)
_EXE=/usr/bin/login(19)
*_CMDLINE=/bin/login -p --(25)
_SYSTEMD_CGROUP=/system.slice/system-getty.slice/getty@tty3.service(67)
_SYSTEMD_UNIT=getty@tty3.service(32)
_SYSTEMD_SLICE=system-getty.slice(33)
MESSAGE=pam_unix(login:auth): check pass; user unknown(54)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
SYSLOG_PID=6074(15)
SYSLOG_TIMESTAMP=Sep  6 13:06:23 (33)
_PID=6074(9)
_SYSTEMD_INVOCATION_ID=15863457ae92465abc17faceb39ecea2(55)
_SOURCE_REALTIME_TIMESTAMP=1693976783617809(43)
-----------------------

9月 06 13:06:23 vm login[6074]: pam_unix(login:auth): authentication failure; logname=LOGIN uid=0 euid=0 tty=/dev/tty3 ruser= rhost=

PRIORITY=5(10)
_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_CAP_EFFECTIVE=1ffffffffff(26)
_TRANSPORT=syslog(17)
SYSLOG_FACILITY=10(18)
SYSLOG_IDENTIFIER=login(23)
_COMM=login(11)
_EXE=/usr/bin/login(19)
*_CMDLINE=/bin/login -p --(25)
_SYSTEMD_CGROUP=/system.slice/system-getty.slice/getty@tty3.service(67)
_SYSTEMD_UNIT=getty@tty3.service(32)
_SYSTEMD_SLICE=system-getty.slice(33)
MESSAGE=pam_unix(login:auth): authentication failure; logname=LOGIN uid=0 euid=0 tty=/dev/tty3 ruser= rhost=(108)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
SYSLOG_PID=6074(15)
SYSLOG_TIMESTAMP=Sep  6 13:06:23 (33)
_PID=6074(9)
_SYSTEMD_INVOCATION_ID=15863457ae92465abc17faceb39ecea2(55)
SYSLOG_RAW=<85>Sep  6 13:06:23 login[6074]: pam_unix(login:auth): authentication failure; logname=LOGIN uid=0 euid=0 tty=/dev/tty3 ruser= rhost= (145)
_SOURCE_REALTIME_TIMESTAMP=1693976783617864(43)
-----------------------

9月 06 13:06:26 vm login[6074]: FAILED LOGIN (1) on '/dev/tty3' FOR 'UNKNOWN', Authentication failure

PRIORITY=5(10)
_MACHINE_ID=e220885a68fa4ab2aceec739ffba2683(44)
_HOSTNAME=vm(12)
_UID=0(6)
_GID=0(6)
_SELINUX_CONTEXT=unconfined
(28)
_CAP_EFFECTIVE=1ffffffffff(26)
_TRANSPORT=syslog(17)
SYSLOG_FACILITY=10(18)
SYSLOG_IDENTIFIER=login(23)
_COMM=login(11)
_EXE=/usr/bin/login(19)
*_CMDLINE=/bin/login -p --(25)
_SYSTEMD_CGROUP=/system.slice/system-getty.slice/getty@tty3.service(67)
_SYSTEMD_UNIT=getty@tty3.service(32)
_SYSTEMD_SLICE=system-getty.slice(33)
MESSAGE=FAILED LOGIN (1) on '/dev/tty3' FOR 'UNKNOWN', Authentication failure(77)
_BOOT_ID=c32dacecd9a34607abdbd577dbde9147(41)
SYSLOG_PID=6074(15)
_PID=6074(9)
_SYSTEMD_INVOCATION_ID=15863457ae92465abc17faceb39ecea2(55)
SYSLOG_TIMESTAMP=Sep  6 13:06:26 (33)
_SOURCE_REALTIME_TIMESTAMP=1693976786726029(43)
-----------------------
